### PR TITLE
Weekly reports marcel

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: weekly_reports/2016_38/marcellii.tex [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Erstellen des ersten Wireframes für die Weboberfläche  &Ja                   \\
+    \rowcolor{OliveGreen!15}
+    Abgabe des Lasten- und Pflichtenhefts                  &Ja                   \\
+    \rowcolor{OliveGreen!15}
+    Datawarehouse beschrieben                              &Ja                   \\
+    \bottomrule[2pt]
+
```
Comment (bastelfreak) [inline weekly_reports/2016_38/marcellii.tex]: please remove the whitespace here

File: weekly_reports/2016_40/marcellii.tex [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Anpassungen an Grafana für den Protoyp               &Ja                    \\
+    \rowcolor{OliveGreen!15}
+    Vorstellen des Prototyps                             &Ja                    \\
+    \rowcolor{OliveGreen!15}
+    Selbststudium der Machbarkeit mit Grafana            &Ja                    \\
+    \rowcolor{OliveGreen!15}
```
Comment (bastelfreak) [inline weekly_reports/2016_40/marcellii.tex]: please remove useless \rowcolor and newline

File: weekly_reports/2016_41/marcellii.tex [hunk 3]
```diff
@@ -0,0 +1,19 @@
+\begin{tabularx}{\textwidth}{Xc}
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Urlaub
+
```
Comment (bastelfreak) [inline weekly_reports/2016_41/marcellii.tex]: drop the whitespace

File: weekly_reports/2016_42/marcellii.tex [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+\vspace{1cm}
+%
+\begin{tabularx}{\textwidth}{Xc}
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Ziele für die nächste Woche}              &                      \\
+    \midrule[2pt]
+    Erstellen der Kriterien zur Evaluierung der Datenbank  &                      \\
+    \rowcolor{OliveGreen!15}
+    Planen des nächstens Sprints                           &                      \\
```
Comment (bastelfreak) [inline weekly_reports/2016_42/marcellii.tex:19]: \bottomrule[2pt] missing?

File: weekly_reports/2016_43/marcellii.tex [hunk 5]
```diff
@@ -0,0 +1,21 @@
+\begin{tabularx}{\textwidth}{Xc}
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Erstellen der Kriterien zu Evaluierung der Datenbank  &Ja                    \\
+    \rowcolor{OliveGreen!15}
+    Planung des nächsten Sprints                          &Ja                    \\
+
```
Comment (bastelfreak) [inline weekly_reports/2016_43/marcellii.tex]: drop whitespace

File: weekly_reports/2016_45/marcellii.tex [hunk 6]
```diff
@@ -0,0 +1,23 @@
+\begin{tabularx}{\textwidth}{Xc}
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Gegenlesen des deutschen/englischen Lasten- und Pflichtenhefts  &Ja                    \\
+    \rowcolor{OliveGreen!15}
+    Kickoff Meeting                      &Ja                    \\
+
```
Comment (bastelfreak) [inline weekly_reports/2016_45/marcellii.tex:9]: drop whitespace

File: weekly_reports/2016_42/marcellii.tex (hunk 4 above)
Comment (Marcellii) [reply weekly_reports/2016_42/marcellii.tex:19]: Hi @bastelfreak i think there is no \bottomrule[2pt] missing. I checked this file and check other weekly reports and i think already is fine on this weekly report from week 42

File: weekly_reports/2016_44/marcellii.tex [hunk 7]
```diff
@@ -0,0 +1,23 @@
+\begin{tabularx}{\textwidth}{Xc}
+    \arrayrulecolor{OliveGreen}
+    \toprule
+    {\bfseries Zu erreichende Ziele der aktuellen Woche} & {\bfseries Erreicht} \\
+    \midrule[2pt]
+    Gegenlesen des deutschen/englischen Lasten- und Pflichtenhefts  &Ja                    \\
+    \rowcolor{OliveGreen!15}
+    Kickoff Meeting                      &Ja                    \\
+
```
Comment (bastelfreak) [inline weekly_reports/2016_44/marcellii.tex:9]: drop whitespace

File: weekly_reports/2016_42/marcellii.tex (hunk 4 above)
Comment (bastelfreak) [reply weekly_reports/2016_42/marcellii.tex:19]: ah, correct

File: weekly_reports/2016_45/marcellii.tex (hunk 6 above)
Comment (Marcellii) [reply weekly_reports/2016_45/marcellii.tex:9]: Please ignor this Weekly Report. We are at the Moment in Week 44 :D

File: weekly_reports/2016_44/marcellii.tex (hunk 7 above)
Comment (Marcellii) [reply weekly_reports/2016_44/marcellii.tex:9]: Here the same. At the end of this week i will provide the correct one the git.

**Pull request merged**